### ISSUE TITLE
Dragging Tab Out from TST to Open as New Window Doesn't Work with Dual Monitors in Stacked Orientation

### ISSUE THREAD

**Issue by ClarkHenderson** [piroor/treestyletab#1613]
## Short description
I am running a three monitor setup, one is mounted above the main monitor, and the third is off to the side of the main.

When dragging out a tab to open as a new window from the main (or side) monitor to the monitor mounted above, nothing happens. No new window is created. This also occurs in the opposite scenario (dragging tab from the top to bottom).

Dragging out a tab to create a new window within the same monitor, or even to the monitors oriented side by side function as expected. I only experience this issue if I drag a tab to the monitor mounted above.

I've only experienced this when dragging the tab to open a new window. Dragging a tab to an existing Firefox window works as expected (on 2.2.11.5829).

## Steps to reproduce

If you have dual monitors, setup the orientation in Windows Display setting so one monitor is stacked above the other. Make sure to select the extend desktop option, rather than duplicate.
 1. Start Firefox with clean profile.
 2. Install TST.
 3. Start Firefox and drag a tab from TST to the monitor mounted above (or below) the Firefox window.
 

## Expected result
The dragged tab should open as a new window

## Actual result
Nothing happens. No new window is created, and the tab still exists in its original place.

## Environment

 * Platform (OS):  Windows 10 x64 Latest Update
 * Version of Firefox: v57 and Developer v58
 * Version (or revision) of Tree Style Tab: 2.2.11.5829 and 2.2.11

**Comment by piroor** [comment]
After more research, I've realized that it is not a problem from multiple monitors. Simply, the tab detaching operation succeeds if you drag the tab to straight left or right. Otherwise - up, bottom, up-left, up-right, down-left, and down-right operations fail. Finally I noticed that the operation fails if the last drag-over target is acceptable for dropping of a tab.

So, by recent commits I've introduced invisible undroppable area around the sidebar contents. I think the problem is now fixed.
